examples/basic/BasicWithAmpBody/BasicWithAmpBody.go

```diff
@@ -3,9 +3,9 @@ package main
 import (
 	"fmt"
 
-	exampleconfig "github.com/socketlabs/socketlabs-go/examples"
-	"github.com/socketlabs/socketlabs-go/injectionapi"
-	"github.com/socketlabs/socketlabs-go/injectionapi/message"
+	exampleconfig "github.com/PraneethChandraThota/socketlabs-go/examples"
```

Comment:
after your tests are complete please return the imports back to the main repo

Reply:
Changed back to the main repo